src/components/Layout.tsx

```diff
@@ -43,20 +44,14 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
   ])
 
   const router = useRouter()
+  const mdScreen = useMedia("(max-width: 768px)")
 
   return (
     <AppProviders>
-      <div className="max-md:hidden">
-        {/* desktop toaster */}
-        <Toaster
-          position="top-right"
-          containerClassName="mr-[2.5%] -mt-[5px]"
-        />
-      </div>
-      <div className="md:hidden">
-        {/* mobile toaster */}
-        <Toaster position="bottom-center" />
-      </div>
+      <Toaster
```

Comment:
The previous method using the className introduces a bug (toast position overlaps)